SwiftWisdom/Core/DirectoryManager/DirectoryManager.swift

```diff
@@ -11,7 +11,10 @@ import Foundation
 public final class DirectoryManager {
     
     enum Error : ErrorType {
-        case UnableToCreatePath(url: NSURL?) // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. Review to see if this is an appropriate change.
+        // TODO: When upgrading to 2.3 Swift, this type changed from NSURL -> NSURL?. 
```

Comment:
We made this change in very much of a hurry on Friday as we needed it for a release. The change was going to go out as is, with very little other eyes on it. The root of the matter is that DirectoryManager can fail in many other ways other than just 'UnableToCreatePath'. I think probably the best thing to do is to create the proper error states for them, rather than grouping them under 'UnableToCreatePath'...hence this comment.